example_site/src/components/ConfigurationControls.js

```diff
@@ -84,9 +69,20 @@ export default function ConfigurationControls({ isOpen }) {
     return orientation ? 'right' : 'left';
   }
 
-  const onSubmit = async (formData) => {
-    const configuration = getConfiguration(formData);
+  function clickButton (config) {
+    if (config.id==='custom') {
+      setShowCustomForm(!showCustomForm);
+      return;
+    }
+    onSelectConfiguration(config.configuration);   
+  }
+
+  async function onSubmit (formData) {
```

Comment:
You don't need this function - you can call `onSelectConfiguration` directly in the `handleSubmit` function below!